include/nuttx/fs/fs.h

```diff
@@ -179,7 +179,8 @@
  * the open method will redirect to another driver and return a file
  * descriptor instead.
  *
- * This case is when SUSv1 pseudo-terminals are used (CONFIG_PSEUDOTERM_SUSV1=y).
+ * This case is when SUSv1 pseudo-terminals are usedi
```

Comment:
usedi->used?

Reply:
That happens when I use vim.  i = INSERT mode.

Reply:
I will fix that.

Reply:
See PR #778

Reply:
oops, sorry.
yes, i'm using vi. and my "i" key has a bad chattering. i plan to fix the keyboard.